api/cpp/tests/manual/platform_qt/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(platform_qt main.cpp)
+target_link_libraries(platform_qt PRIVATE Slint::Slint Qt::Gui Qt::Widgets Qt::GuiPrivate)
```

Comment:
I suppose widgets are not really needed, are they? I see that the C++ code creates a `QApplication`, but I think a `QGuiApplication` should also suffice.

Reply:
The problem is that, by default, we're going to us the native style and that needs a QApplication and assert with a QGuiApplication

Reply:
That's a compelling argument :)